README.md

```diff
@@ -67,16 +67,16 @@ How to deploy this infrastructure is covered in a [later section](#deploying-inf
 ### Local setup
 
 1. Clone the repository locally (or create a new repo from this template)
```

Comment:
You can leave the numbers all as `1.`. When they are rendered, they will automatically increment